lib/models/notification.ts

```diff
@@ -69,22 +70,39 @@ export const EmailNotification = t.interface({
 });
 export type EmailNotification = t.TypeOf<typeof EmailNotification>;
 
+// Webhook Notification
+
+export const NotificationChannelWebhook = t.interface({
+  url: WebUrl
+});
+export type NotificationChannelWebhook = t.TypeOf<
+  typeof NotificationChannelWebhook
+>;
+
+export const WebhookNotification = t.interface({
+  ...NotificationBase.props,
+  channels: t.interface({
```

Comment:
I don't understand why we need `channels: {}` with a single key in it

Reply:
renamed